easybuild/easyblocks/generic/binary.py

```diff
@@ -58,6 +58,9 @@ def extra_options(extra_vars=None):
             'install_cmd': [None, "Install command to be used.", CUSTOM],
             # staged installation can help with the hard (potentially faulty) check on available disk space
             'staged_install': [False, "Perform staged installation via subdirectory of build directory", CUSTOM],
+            'prepend_to_path': ['',
+                                "Prepend the given path to environment PATH in the module file. Set to None to skip",
```

Comment:
Please clarify that the specified path is relative to the installation prefix? Also, putting each part on a separate line is overkill?

```python
'prepend_to_path': ['', "Prepend given directory (relative to installation prefix) to $PATH (use None to skip)",
                    CUSTOM],

Reply:
Also, maybe we should make the default value `['']` (and adjust accordingly below), to allow for adding multiple location to `$PATH`?

And then you don't need to use `None` as a magic value, an empty list will do too to skip adding anything to `$PATH`...

Reply:
The list idea is good, I'm just not sure if it is helpful. Can there be any situation where you would want to add multiple paths to PATH? Usually  it is "bin" and this already lets you add one additional path. Should be enough but I'll change that if you like it better. And I don't consider `None` to be a magic value. IMO it is very descriptive.

About the separate lines: The Code-review bot was complaining about line length. If I add the relative part then even this would not fit. Where and how shall I split it? I could not find convention.

Reply:
I admit that having to add multiple locations to `$PATH` is rare, but it does happen (e.g. both `bin` & `scripts`). You are right that not being able to provide multiple locations via `prepend_to_path` is not a big issue though, you can always get additional locations included via `modextrapaths = {'PATH': [...]}` in the easyconfig file.
I was only referring to `None` as magical value since an empty string `''` can't be used to skip adding a location to `$PATH`, and with a list this is no longer necessary (since an empty list `[]` also does that).
With that in mind, let's indeed keep the default value as is (`''`).

W.r.t. line length:

```python
'prepend_to_path': ['', "Prepend given directory (relative to installation prefix) to $PATH"
                    " (use None to skip)", CUSTOM],
```

(not using a `,` after the first line results in the two strings to be concatenated for the help message)